HLTrigger/Configuration/python/customizeHLTTrackingForPhaseI2017.py

```diff
@@ -106,130 +106,268 @@ def customizeHLTForPFTrackingPhaseI2017(process):
-        seedPairPenalty = cms.int32(0),
-        strictSeedExtension = cms.bool(True)
+
+ 
+    process.hltIter1PFlowPixelSeeds = cms.EDProducer( "SeedGeneratorFromProtoTracksEDProducer",
+        useEventsWithNoVertex = cms.bool( True ),
+        originHalfLength = cms.double( 0.3 ), 
+        useProtoTrackKinematics = cms.bool( False ),
+        usePV = cms.bool( True ),
+        SeedCreatorPSet = cms.PSet(  refToPSet_ = cms.string( "HLTSeedFromProtoTracks" ) ),
+        InputVertexCollection = cms.InputTag( "hltTrimmedPixelVertices" ),
+        TTRHBuilder = cms.string( "hltESPTTRHBuilderPixelOnly" ),
+        InputCollection = cms.InputTag( "hltIter1PixelTracks" ),
+        originRadius = cms.double( 0.1 )
+    )
+    process.HLTIter1PSetTrajectoryBuilderIT.strictSeedExtension = cms.bool(True)
+  
+
+    process.HLTIter1PSetTrajectoryFilterIT = cms.PSet( 
+	    ComponentType = cms.string('CkfBaseTrajectoryFilter'),
+	    chargeSignificance = cms.double(-1.0),
+	    constantValueForLostHitsFractionFilter = cms.double(2.0),
+	    extraNumberOfHitsBeforeTheFirstLoop = cms.int32(4),
+	    maxCCCLostHits = cms.int32(0),
+	    maxConsecLostHits = cms.int32(1),
+	    maxLostHits = cms.int32(1),  
+	    maxLostHitsFraction = cms.double(0.1),
+	    maxNumberOfHits = cms.int32(100),
+	    minGoodStripCharge = cms.PSet( refToPSet_ = cms.string( "HLTSiStripClusterChargeCutNone" ) ),
+	    minHitsMinPt = cms.int32(3),
+	     minNumberOfHitsForLoopers = cms.int32(13),
+	    minNumberOfHitsPerLoop = cms.int32(4),
+	    minPt = cms.double(0.2),
+	    minimumNumberOfHits = cms.int32(4), 
+	    nSigmaMinPt = cms.double(5.0),
+	    pixelSeedExtension = cms.bool(True),
+	    seedExtension = cms.int32(1),
+	    seedPairPenalty = cms.int32(0),
+	    strictSeedExtension = cms.bool(True)
     )
 
-    process.HLTIter1PSetTrajectoryBuilderIT = cms.PSet(
-        inOutTrajectoryFilter = cms.PSet( refToPSet_ = cms.string('HLTIter1PSetTrajectoryFilterInOutIT') ),
-        propagatorAlong = cms.string( "PropagatorWithMaterialParabolicMf" ),
-        trajectoryFilter = cms.PSet(  refToPSet_ = cms.string( "HLTIter1PSetTrajectoryFilterIT" ) ),
-        maxCand = cms.int32( 2 ),
-        ComponentType = cms.string( "CkfTrajectoryBuilder" ),
-        propagatorOpposite = cms.string( "PropagatorWithMaterialParabolicMfOpposite" ),
-        MeasurementTrackerName = cms.string( "hltIter1ESPMeasurementTracker" ),
-        estimator = cms.string( "hltESPChi2ChargeMeasurementEstimator16" ),
-        TTRHBuilder = cms.string( "hltESPTTRHBWithTrackAngle" ),
-        updator = cms.string( "hltESPKFUpdator" ),
-        alwaysUseInvalidHits = cms.bool( False ),
-        intermediateCleaning = cms.bool( True ),
-        lostHitPenalty = cms.double( 30.0 ),
-        useSameTrajFilter = cms.bool(False) # new ! other iteration should have it set to True
+    process.HLTIter1PSetTrajectoryFilterInOutIT = cms.PSet
+	ComponentType = cms.string('CkfBaseTrajectoryFilter'),
```

Comment:
There is a "(" missing on the line with cms.PSet.
It is good practice to actually test you code before making a PR.